contrib/dashboards-gateway.Dockerfile

```
@@ -0,0 +1,13 @@
+FROM maven:3-amazoncorretto-21 as build
+WORKDIR /work
+COPY . /work/
+RUN mvn -f /work/contrib/dashboards-gateway/pom.xml -DskipTests package
+
+FROM amazoncorretto:21
+COPY --from=build /work/contrib/dashboards-gateway/target/tools-dashboards-gateway.jar /dashboards-gateway.jar
+# Current amazoncorretto:21 images appear to include curl, but install it explicitly so the
+# healthcheck remains stable across base image refreshes.
+RUN yum install -y curl && yum clean all
```

Hadolint (2.14.0)

[warning] Specify version with `yum install -y <package>-<version>`.

(DL3033)